src/components/RoarDataTable.vue

```diff
@@ -424,31 +424,59 @@ let toolTipByHeader = (header) => {
   return '';
 };
 
+function getIndexTask(colData, task){
+  for (let index = 0; index < colData.assignment.assessments.length; index++) {
+    if(colData.assignment.assessments[index].taskId === task){
+      return index;
+    }
+  }
+};
+
 let returnScoreTooltip = (colHeader, colData) => {
   let toolTip = '';
-
+  let index = -1;
   if (colHeader === 'Phoneme' && colData.scores?.pa?.standard) {
     toolTip += colData.scores.pa?.support_level + '\n' + '\n';
     toolTip += 'Percentile: ' + colData.scores?.pa?.percentile + '\n';
     toolTip += 'Raw Score: ' + colData.scores?.pa?.raw + '\n';
     toolTip += 'Standardized Score: ' + colData.scores?.pa?.standard + '\n';
+    index = getIndexTask(colData, 'pa');
+    if(colData.assignment.assessments[index].engagementFlags !== undefined && colData.assignment.assessments[index].engagementFlags !== '' && !colData.assignment.assessments[index].reliable){
+      toolTip += '\n' + 'Reliability: ' + Object.keys(colData.assignment.assessments[index].engagementFlags) + '\n';
+    }
```

Comment:
I'd suggest here:
- Saving the flags to a variable for readability
- Converting the camel cased flags to lower case and separating by commas and spaces

```suggestion
    const flags = colData.assignment.assessments[index].engagementFlags;
    if(flags !== undefined && flags !== '' && !colData.assignment.assessments[index].reliable){
      toolTip += '\n' + 'Reliability: ' + Object.keys(flags).map(_lowerCase).join(', ') + '\n';
    }
```

This suggestion uses [lodash's `lowerCase` method](https://lodash.com/docs/4.17.15#lowerCase), so you'll have to import that above as well.